#! /bin/sh /usr/share/dpatch/dpatch-run
## webxml.dpatch by  <jetty@packaging-u1040-32.dev.opengeo.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: adding placeholder for PROXY_BASE_URL and setting 
## DP: GEOSERVER_DATA_DIRECTORY

@DPATCH@

diff -rc opengeo-geoserver-orig/geoserver/WEB-INF/web.xml opengeo-geoserver/geoserver/WEB-INF/web.xml
*** opengeo-geoserver-orig/geoserver/WEB-INF/web.xml	2011-01-20 00:02:59.000000000 +0000
--- opengeo-geoserver/geoserver/WEB-INF/web.xml	2011-01-20 00:05:50.000000000 +0000
***************
*** 40,58 ****
      <param-value>50</param-value>
    </context-param>
    
-     <!-- 
      <context-param>
        <param-name>PROXY_BASE_URL</param-name>
!       <param-value>http://82.58.146.45/</param-value>
      </context-param>
-      -->
     
-      <!--
      <context-param>
         <param-name>GEOSERVER_DATA_DIR</param-name>
!         <param-value>C:\eclipse\workspace\geoserver_trunk\cite\confCiteWFSPostGIS</param-value>
      </context-param> 
-    -->
      
      <!-- pick up all spring application contexts -->
      <context-param>
--- 40,54 ----
      <param-value>50</param-value>
    </context-param>
    
      <context-param>
        <param-name>PROXY_BASE_URL</param-name>
!       <param-value>http://MYHOST</param-value>
      </context-param>
     
      <context-param>
         <param-name>GEOSERVER_DATA_DIR</param-name>
!        <param-value>/usr/share/opengeo-suite-data/geoserver_data</param-value>
      </context-param> 
      
      <!-- pick up all spring application contexts -->
      <context-param>
